package.json

```diff
@@ -64,6 +64,8 @@
     "file-saver": "^1.3.8",
     "intl": "~1.2.5",
     "jquery": "~3.2.1",
+    "lodash.foreach": "^4.5.0",
+    "lodash.map": "^4.6.0",
```

Comment:
is there any reason why to continue using lodash map and lodash foreach? I believe there are es6 compatible versions of those methods.. 

https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_each
https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_map

Reply:
orderby is not so simple though and used by our sortabular/table sorting component ;(